package.json

```diff
@@ -16,7 +17,9 @@
     "generator:docs": "turbo run docs --filter=@asyncapi/generator",
     "generator:docker:build": "turbo run docker:build --filter=@asyncapi/generator",
     "generator:lint": "turbo run lint --filter=@asyncapi/generator",
-    "generator:lint:tpl:validator": "turbo run lint:tpl:validator --filter=@asyncapi/generator"
+    "generator:lint:tpl:validator": "turbo run lint:tpl:validator --filter=@asyncapi/generator",
+    "generator:update:snapshot": "turbo run test:integration:update --filter=@asyncapi/generator",
+    "nunjucks-filters:test": "turbo run test --filter=@asyncapi/generator-filters"
```

Comment:
is it correct? I think tests will not trigger as `@asyncapi/nunjucks-filters` is what is in the package.json

Reply:
this is fine this is not for running in ci. when we do npm run test turborepo will go through all the package it have and run the test if they have test script in package.json. this one is only for testing manually. in local.

Reply:
no this is not correct. fixed it.